addons/website/tests/test_page.py

```diff
@@ -228,3 +229,23 @@ def test_unpublished_page(self):
         r = self.url_open(specific_page.url)
         self.assertEqual(r.status_code, 200, "Admin should see the specific unpublished page")
         self.assertEqual('I am a specific page' in r.text, True, "Admin should see the specific unpublished page")
+
+    def test_search(self):
+        dbname = common.get_db_name()
+        admin_uid = self.env.ref('base.user_admin').id
+        website = self.env['website'].get_current_website()
+
+        robot = self.xmlrpc_object.execute(
```

Comment:
why test in xmlrpc ? 

can we simple do

```
self.authenticate('admin', 'admin')
robot = self.env['website'].with_context(website=website.id).search_pages('page')
```

just to understand ? And learn for my future test

Reply:
As I was debugging from the client I think I just went "routing -> http -> needs request -> xmlrpc" and didn't even think to use straight RPC access.